docker/kites/net-tests-dev/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:10
```

Comment:
You can greatly simplify this if you use a single Dockerfile and multiple build steps.

The steps are going to be the following:

1. FROM debian:10, install iperf3 and netsniff-ng, expose 5201, set entrypoint and CMD
1. FROM step 1, install nano, tcpdump and screen

And you can keep the Dockerfile in docker/kites/net-tests.

Reply:
@ferrarimarco this is very interesting, and it is ok to produce net-test-dev container
our requirement is also to produce a net-test container that only includes step 1
the simple solution is to have two Dockerfile
kites/net-test-dev/Dokerfile   (step 1 and step 2)
kites/net-test/Dokerfile   (only step 1)
for me it would be OK, but if you have better solutions let us know

Reply:
I see how it works and I do it. Thanks for the tips Marco!

Reply:
So, with a multistage build, you can have a single dockerfile and build multiple containers on top of each other. That's what I was suggesting